src/bindings/RescriptCompilerApi.res

```diff
@@ -431,6 +437,7 @@ module Compiler = {
     }
 
     Belt.Option.forEach(moduleSystem, moduleSystem => t->setModuleSystem(moduleSystem)->ignore)
+    Belt.Option.forEach(config.open_modules, modules => t->setOpenModules(modules)->ignore)
```

Comment:
I guess here one needs to check for compiler version to be V4 as well?

Reply:
Did that but at a different spot (see [commit](https://github.com/rescript-association/rescript-lang.org/pull/729/commits/32cfe99654f3954570f9290770787968d0454160)).